src/librbd/ImageWatcher.cc

```diff
@@ -149,7 +149,7 @@ template <typename I>
 void ImageWatcher<I>::notify_async_complete(const AsyncRequestId &request,
                                             int r) {
   ldout(m_image_ctx.cct, 20) << this << " remote async request finished: "
-			     << request << " = " << r << dendl;
+			     << request << ", r=" << r << dendl;
```

Comment:
The existing format is consistent with how progress is reported (`: [4164,140018551113088,1] @ 1/10`) and how completion is reported on the other side (`: [4164,140018551113088,1]=0`, see `ImageWatcher::handle_payload()` for `AsyncCompletePayload`).  If anything, I'd just remove spaces around `=` here.

Reply:
is that the only place where we don't describe as ", r=" ? If so, will it make sense to fix it rather than making this look like  `[ request-ID ] = ret` ?

Reply:
I'm not sure if it's the only place, but if you want to keep spaces (i.e. leave `ImageWatcher::notify_async_complete()` intact), I would ask to add spaces here:
https://github.com/ceph/ceph/blob/af3682ffd858d3678ede83075e5188117278c934/src/librbd/ImageWatcher.cc#L1094-L1105